lib/ObisNames.js

```diff
@@ -6,352 +6,352 @@
         },
         "nl": {
             0: "Algemeen doel",
-            1: "Totaal aankoop actieve energie +",
-            2: "Totaal afgegeven actieve energie -",
-            3: "Totale aankoop reactieve energie +",
-            4: "Totale reactieve energie-output -",
-            5: "Totale reactieve energie Q I",
-            6: "Totale reactieve energie Q II",
-            7: "Totale reactieve energie Q III",
-            8: "Totale reactieve energie Q IV",
-            9: "Som schijnbare stroomafname +",
-            10: "Totaal schijnbaar vermogen -",
-            11: "Actueel",
+            1: "Totale werklastenergie afname +",
+            2: "Totale werklastenergie teruggave -",
+            3: "Totale blindlastenergie afname +",
+            4: "Totale blindlastenergie teruggave -",
+            5: "Totale blindlastenergie Q I",
+            6: "Totale blindlastenergie Q II",
+            7: "Totale blindlastenergie Q III",
+            8: "Totale blindlastenergie Q IV",
+            9: "Totale schijnbelasting afname +",
+            10: "Totale schijnbelasting teruggave -",
+            11: "Stroom",
             12: 'Voltage',
-            13: "Vermogensfactor",
-            14: 'Frequency',
-            16: "Totaal actief vermogen",
-            21: "L1 referentie actieve energie +",
-            22: "L1 uitgang van actieve energie -",
-            23: "L1-referentie reactieve energie +",
-            24: "L1 uitgang reactieve energie -",
-            25: "L1 reactieve energie Q I",
-            26: "L1 reactieve energie Q II",
-            27: "L1 reactieve energie Q III",
-            28: "L1 Reactieve energie Q IV",
-            29: "L1 schijnbaar vermogen referentie +",
-            30: "L1 schijnbaar vermogen -",
+            13: "Prestatiefactor",
+            14: 'Frequentie',
+            16: "Totaalprestatie",
+            21: "L1 werklastenergie afname +",
+            22: "L1 werklastenergie teruggave -",
+            23: "L1-blindlastenergie afname +",
+            24: "L1 blindlastenergie teruggave -",
+            25: "L1 blindlastenergie Q I",
+            26: "L1 blindlastenergie Q II",
+            27: "L1 blindlastenergie Q III",
+            28: "L1 blindlastenergie Q IV",
+            29: "L1 schijnbelasting afname +",
+            30: "L1 schijnbelasting teruggave -",
             31: "L1 Stroom",
-            32: "L1 Spanning",
-            33: "L1 Vermogensfactor",
-            41: "L2 referentie actieve energie +",
-            42: "L2 actieve energie output -",
-            43: "L2-referentie reactieve energie +",
-            44: "L2 uitgang reactieve energie -",
-            45: "L2 reactieve energie Q I",
-            46: "L2 reactieve energie Q II",
-            47: "L2 reactieve energie Q III",
-            48: "L2 reactief vermogen Q IV",
-            49: "L2 schijnbaar vermogen referentie +",
-            50: "L2 schijnbaar vermogen -",
+            32: "L1 Voltage",
```

Comment:
Discutabel :) voor de leek voltage in he electro vak spanning ik ben het eens met Voltage is duidelijker